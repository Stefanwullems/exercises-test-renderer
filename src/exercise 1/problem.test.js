import React from 'react'
import { create } from 'react-test-renderer'


const Component = (props) => {
  return <div>Hello</div>
}

// Find a method on testRenderer that you can use to make the test pass
// reference of all methods https://reactjs.org/docs/test-renderer.html

test('the component hasn\'t changed', () => {
  const {/* code here */            /* end */ } = create(<Component />)

  let component
  /* code here */

  /* end */

  expect(component).toMatchSnapshot()
})