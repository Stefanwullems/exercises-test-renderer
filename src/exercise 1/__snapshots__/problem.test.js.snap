// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the component hasn't changed 1`] = `
<div>
  Hello
</div>
`;
